diff --git a/node_modules/react-awesome-button/index.d.ts b/node_modules/react-awesome-button/index.d.ts
index 72ccf32..b583626 100644
--- a/node_modules/react-awesome-button/index.d.ts
+++ b/node_modules/react-awesome-button/index.d.ts
@@ -31,8 +31,8 @@ declare module 'react-awesome-button' {
     visible: boolean;
   }
 
-  export default class AwesomeButton extends Component<
-    AwesomeButtonProps,
+  export class AwesomeButton extends Component<
+    Partial<AwesomeButtonProps>,
     {}
   > {}
 
@@ -51,8 +51,8 @@ declare module 'react-awesome-button' {
     type: string;
   }
 
-  export default class AwesomeButtonProgress extends Component<
-    AwesomeButtonProgressProps,
+  export class AwesomeButtonProgress extends Component<
+    Partial<AwesomeButtonProgressProps>,
     {}
   > {}
 
@@ -73,8 +73,8 @@ declare module 'react-awesome-button' {
     user: string;
   }
 
-  export default class AwesomeButtonSocial extends Component<
-    AwesomeButtonSocialProps,
+  export class AwesomeButtonSocial extends Component<
+  Partial<AwesomeButtonSocialProps>,
     {}
   > {}
 }
